shenyu-plugin/shenyu-plugin-divide/src/main/java/org/apache/shenyu/plugin/divide/DividePlugin.java

```diff
@@ -110,8 +114,15 @@ protected Mono<Void> doExecute(final ServerWebExchange exchange, final ShenyuPlu
         exchange.getAttributes().put(Constants.RETRY_STRATEGY, StringUtils.defaultString(ruleHandle.getRetryStrategy(), RetryEnum.CURRENT.getName()));
         exchange.getAttributes().put(Constants.LOAD_BALANCE, StringUtils.defaultString(ruleHandle.getLoadBalance(), LoadBalanceEnum.RANDOM.getName()));
         exchange.getAttributes().put(Constants.DIVIDE_SELECTOR_ID, selector.getId());
-        return ruleHandle.getLoadBalance().equals(P2C) ? chain.execute(exchange).doOnSuccess(e -> responseTrigger(upstream
-        )).doOnError(throwable -> responseTrigger(upstream)) : chain.execute(exchange);
+        if (ruleHandle.getLoadBalance().equals(P2C)) {
```

Comment:
whats mean ?

Reply:
Different strategies require different methods to be implemented.Does it need improvement？